docs/pages/build/eas-json.md

```diff
@@ -106,13 +110,40 @@ If you'd like to build a debug APK use the following example. Also, make sure th
 
 ### Managed Project
 
-Managed projects are not supported at the moment but the profile's schema is going to look like this:
+The schema of a build profile for a managed Android project looks like this:
 
 ```json
 {
   "workflow": "managed",
   "credentialsSource": "local" | "remote" | "auto", // default: "auto"
-  "buildType": "app-bundle" | "apk" // default: "app-bundle"
+  "buildType": "app-bundle" | "apk", // default: "app-bundle"
+  "releaseChannel": string, // default: "default"
+  "distribution": "store" | "internal" // default: "store"
+}
+```
+
+- `"workflow": "managed"` indicates that your project is a managed one.
+- `credentialsSource` defines the source of credentials for this build profile. If you want to take advantage of your own `credentials.json` file, set this to `local` ([learn more on this here](advanced-credentials-configuration.md)). If you want to use the credentials Expo already has stored for you, choose `remote`. If you're not sure what to do but you probably won't be running builds from a CI, choose `auto` (this is the default option).
```

Comment:
```suggestion
- `credentialsSource` defines the source of credentials for this build profile. If you want to provide your own `credentials.json` file, set this to `local` ([learn more on this here](advanced-credentials-configuration.md)). If you want to use the credentials Expo already has stored for you, choose `remote`. If you're not sure what to do, choose `auto` (this is the default option).
```

probably best to just leave this bit about CI out for now, people can figure it out when they look into configuring for CI